### ISSUE TITLE
Fine-tuning

### ISSUE THREAD

**Issue by StefanoUnibo** [weiliu89/caffe#440]
### Issue summary
Fine-tuning on pre-trained models.

### Steps to reproduce
Hi, can you help me to make a fine-tuning on a pre-trained model with a dataset made by me in pascal VOC format?
Usually, in a classic model, I modify the last layer to perform a fine-tuning, but this time an error appears: 

F0210 20:47:05.647130 30300 multibox_loss_layer.cpp:141] Check failed: num_priors * num_classes == bottom[1]->channels() (43660 vs. 183372) Number of priors must match number of confidence predictions.

The number of classes in my dataset is different between the pre-trained model.

Thanks for your advices!

### Your system configuration
Operating system: Ubuntu 15.04 (GNU/Linux 3.19.0-79-generic x86_64)

**Comment by weiliu89** [comment]
Are your classes a subset of VOC classes?

**Comment by StefanoUnibo** [comment]
The classes are different to VOC classes. I had modified the labalmap_voc for my dataset.

**Comment by weiliu89** [comment]
You have to set prefix in the CreatMultiBoxHead to change the confidence prediction layer name.

**Comment by StefanoUnibo** [comment]
Please, can You explain with more details the procedure? Thanks.

**Comment by thhung** [comment]
Maybe I'm late for the party.
Go to this file: https://github.com/weiliu89/caffe/blob/ssd/examples/ssd/ssd_pascal.py 
If you search CreatMultiBoxHead, you will see the function CreatMultiBoxHead at line 440. 
Add conf_postfix="_whatever_you_like" to the parameters of the function, you will change the name of confidence prediction layer like he means above. That's all you need to do.

**Comment by Anhaoxu** [comment]
@thhung Did you mean add"**conf_postfix=43660**" as this above problem, in this case, the Number of priors can match number of confidence predictions?

**Comment by thhung** [comment]
No, what i mean is you could change the name of layers so it will be more convenient for you when you fine-tune. It has nothing to do with the number of confidence predictions.

**Comment by Hzzone** [comment]
I have removed some lines in `ssd_pascal.py` and edited lines blew especially:
```python
mbox_layers = CreateMultiBoxHead(net, data_layer='data', from_layers=mbox_source_layers,
        use_batchnorm=use_batchnorm, min_sizes=min_sizes, max_sizes=max_sizes,
        aspect_ratios=aspect_ratios, steps=steps, normalizations=normalizations,
        num_classes=num_classes, share_location=share_location, flip=flip, clip=clip,
        prior_variance=prior_variance, kernel_size=3, pad=1, lr_mult=lr_mult, conf_postfix='WHATERVER_YOU_LIKE')
```
and then I successfully fine-tuning PASCAL VOC pretrained model via command line tool.
Hopefully it can help you. For more detail reference [generate_model.py](https://github.com/Hzzone/Hand-Keypoint-Detection/blob/master/model/generate_model.py).
